content/working_with/official_plugins/Orchestration/terraform.md

```diff
@@ -86,19 +86,36 @@ This refers to a Terraform Plan module.
 **Operations**
 
   * `terraform.reload`: Reloads the Terraform template given the following inputs:
-    * `source` : the new template location by default the `last_source_location` but it can be changed to be another location or even URL to a new template
-    * `destroy_previous` : boolean if set to True it will trigger destroy for the previously created resources , if False it will keep them and maintain the state file , and terraform will calculate the changes needed to be applied to those already created resources
-  * `terraform.refresh`: Refresh Terraform state file, if any changes were done outside of terraform so it will update the runtimes properties to match the real properties for the created resources
+    * `source`: the new template location by default the `last_source_location` but it can be changed to be another location or even URL to a new template
+    * `destroy_previous`: boolean if set to True it will trigger destroy for the previously created resources , if False it will keep them and maintain the state file , and Terraform will calculate the changes needed to be applied to those already created resources
+  * `terraform.refresh`: Refresh Terraform state file, if any changes were done outside of Terraform so it will update the runtime properties to match the real properties for the created resources under `state` runtime property.
+    Moreover, If there are any drifts between the template and the current state it will be saved under `drifts` runtime property.
```

Comment:
```suggestion
    Moreover, If there are any drifts between the template and the current state it will be saved under the `drifts` runtime property.
```